docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -0,0 +1,255 @@
+Using Public Clouds as backends
+===============================
+
+Introduction
+------------
+
+As stated in our `GETTING STARTED guide <../GETTING_STARTED/#location-configuration>`__,
+new backends can be added by creating a region with the right endpoint and
+credentials.
+This section of the documentation shows you how to set up our currently
+supported public cloud backends:
+
+- `Amazon S3 <#aws-s3-as-a-backend>`__ ;
+- `Microsoft Azure <#microsoft-azure-as-a-backend>`__ .
+
+AWS S3 as a backend
+-------------------
+
+locationConfig.json
+~~~~~~~~~~~~~~~~~~~
+
+Add a new region, which should describe the AWS S3 hosted bucket you'll be
+writing your data to. There are a few configurable options here:
+
+- :code:`type` : set to :code:`aws_s3` to indicate this region is hosted in AWS S3;
+- :code:`legacyAwsBehavior` : set to :code:`true` to indicate this region should
+  behave like AWS S3 :code:`us-east-1` region, set to :code:`false` to indicate
+  this region should behave like any other AWS S3 region;
+- :code:`bucketName` : set to an *existing bucket* in your AWS S3 Account; this
```

Comment:
The user will have to make sure that the bucket both exists and has versioning enabled. I see it mentioned in line 151 but I think it makes more sense to have it mentioned here in the beginning of the guide.

Reply:
I added a section saying "From the AWS Console", and stating what should be done. Please let me know if it's good for you